lib/mail/AttachmentMimeType.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This helper defines the attachment mime types for a /mail/send API call
+ *
+ * PHP Version - 5.6, 7.0, 7.1, 7.2
+ *
+ * @package   SendGrid\Mail
+ * @author    Michael Dennis <michaeljdennis@gmail.com>
+ * @copyright 2018 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+
+namespace SendGrid\Mail;
+
+/**
+ * This class is used to define the attachment mime types for the /mail/send API call
+ *
+ * @package SendGrid\Mail
+ */
+abstract class AttachmentMimeType
+{
+    const APPLICATION_ECMASCRIPT = "application/ecmascript";
```

Comment:
It seems that you want to collect the all available/important mime types?

It seems that some important mime types are not in this abstract class.

Please refer this [completed mime type lists ](https://www.sitepoint.com/mime-types-complete-list/) to complete this lists if possible.

Reply:
@thinkingserious Which MIME types from the above list would you like added to this class?